watertap/unit_models/ion_exchange_0D.py

```diff
@@ -0,0 +1,1352 @@
+            units=pyunits.m,
+            doc="Bed diameter [m]",
+        )
+
+        self.bed_depth = Var(
+            initialize=4, bounds=(0.1, 12), units=pyunits.m, doc="Bed depth [m]"
+        )
+
+        self.bed_area = Var(
+            initialize=0.63,
+            units=pyunits.m**2,
+            doc="Bed area [m2]",
+        )
+
+        self.bed_porosity = Var(
+            initialize=0.5,
+            bounds=(0.45, 0.65),
+            units=pyunits.dimensionless,
+            doc="Bed porosity [-]",
+        )
+
+        self.col_height = Var(
+            initialize=2,
+            bounds=(0.6, 12),
+            units=pyunits.m,
+            doc="Column height [m]",
+        )
+
+        self.col_vol = Var(
+            initialize=10,
+            units=pyunits.m**3,
+            doc="Column volume of one unit [m3]",
+        )
+
+        # ====== Kinetic variables ====== #
+
+        self.partition_ratio = Var(
+            initialize=100,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Partition ratio [-]",
+        )
+
+        self.fluid_mass_transfer_coeff = Var(
+            ion_set,
+            initialize=1e-3,
+            bounds=(0, None),
+            units=pyunits.m / pyunits.s,
+            doc="Fluid mass transfer coefficient [m/s]",
+        )
+
+        self.rate_coeff = Var(
+            ion_set,
+            initialize=0.214e-3,
+            bounds=(0, 1),
+            units=pyunits.m**3 / (pyunits.kg * pyunits.s),
+            doc="Rate coefficient [m3/(kg*s)]",
+        )
+
+        self.t_breakthru = Var(
+            initialize=1e7,
+            # bounds=(0, 4.3E6),  #DOW, ~7 weeks max breakthru time
+            bounds=(0, None),
+            units=pyunits.s,
+            doc="Breakthrough time [s]",
+        )
+
+        self.t_contact = Var(
+            initialize=520,
+            bounds=(120, None),
+            units=pyunits.s,
+            doc="Contact time [s]",
+        )
+
+        self.t_waste = Var(
+            initialize=12,
+            bounds=(1, None),
+            units=pyunits.s,
+            doc="Regen + Rinse + Backwash time [s]",
+        )
+
+        self.num_transfer_units = Var(
+            initialize=1e6,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Number of transfer units [-]",
+        )
+
+        self.HTU = Var(
+            ion_set,
+            initialize=0.5,
+            bounds=(0, 1),
+            units=pyunits.m,
+            doc="Height of a transfer unit",
+        )
+
+        self.dimensionless_time = Var(
+            initialize=1,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="Dimensionless time [-]",
+        )
+
+        self.lh = Var(
+            initialize=0,
+            bounds=(-20, 20),
+            units=pyunits.dimensionless,
+            doc="Left hand side of kinetic equation",
+        )
+
```

Comment:
The meaning of this var is hard to get.

Reply:
Yes, thank you. I will clear up my doc strings in next push.